docs/about.md

```diff
@@ -0,0 +1,49 @@
+## Mailing list
+
+OpenBLAS users: <https://groups.google.com/forum/#!forum/openblas-users>
+
+OpenBLAS developers: <https://groups.google.com/forum/#!forum/openblas-dev>
+
+## Donations
+
+Thank you for the support.
+
+You can read OpenBLAS statement of receipts and disbursement and cash balance on [google doc](https://docs.google.com/spreadsheet/ccc?key=0AghkTjXe2lDndE1UZml0dGpaUzJmZGhvenBZd1F2R1E&usp=sharing).
+
+### Fundraiser
+
+* [2013.8] [Testbed for OpenBLAS project](https://www.bountysource.com/fundraisers/443-testbed-for-openblas-project)
+
+Here is [Backer list](https://github.com/xianyi/OpenBLAS/blob/develop/BACKERS.md).
```

Comment:
Can you update the link to https://github.com/OpenMathLib/OpenBLAS/blob/develop/BACKERS.md?